htdocs/core/ajax/check_notifications.php

```diff
@@ -97,13 +109,29 @@
 
             $actionmod->fetch($obj->id);
 
-            // Message must be formated and translated to be used with javascript directly
+            $actioncommReminder = new ActionCommReminder($db);
```

Comment:
To avoid to init a new object in memory for each pass in loop, move the creation of $actioncommReminder just before the while(). The fetch just after will erase all its data and reuse only 1 object for each pass.